app/models/reports/repository.rb

```diff
@@ -0,0 +1,100 @@
+module Reports
+  class Repository
+    def initialize(regions, periods:)
+      @regions = Array(regions)
```

Comment:
This will enforce that for us -- if you pass in an array it remains an array.  If you pass in one region, it makes it into an array.

I'd like clients to be able to pass either either and not have to worry about, as there are lots of case where we want to do both.